alacritty/src/display/content.rs

```diff
@@ -287,19 +286,9 @@ impl RenderableCell {
     }
 
     /// Apply [`CellRgb`] colors to the cell's colors.
-    fn compute_cell_rgb(
-        cell_fg: &mut Rgb,
-        cell_bg: &mut Rgb,
-        bg_alpha: &mut f32,
-        fg: CellRgb,
-        bg: CellRgb,
-    ) {
+    fn compute_cell_rgb(cell_fg: &mut Rgb, cell_bg: &mut Rgb, fg: CellRgb, bg: CellRgb) {
         let old_fg = mem::replace(cell_fg, fg.color(*cell_fg, *cell_bg));
         *cell_bg = bg.color(old_fg, *cell_bg);
-
-        if bg != CellRgb::CellBackground {
-            *bg_alpha = 1.0;
-        }
```

Comment:
Is there a reason we do this? I understand that if the bg is different from the actual background then we're setting the alpha to 1.0, but this was the main reason why hints, selections, and search matches were opaque

Reply:
Thankyou for the explanation!